themes/crisp/translations/pt_BR.json

```diff
@@ -88,18 +88,18 @@
     "views.classification.class.c": "Nota C",
     "views.classification.class.c.desc": "Os termos de serviço são ok mas algumas questões merecem a sua consideração.",
     "views.classification.class.d": "Nota D",
-    "views.classification.class.d.desc": "Os termos de serviço são muito injustos ou há algumas questões importantes que precisam da sua atenção.",
+    "views.classification.class.d.desc": "Os termos de serviço são muito injustos ou existem algumas questões importantes que precisam da sua atenção.",
     "views.classification.class.e": "Nota E",
     "views.classification.class.e.desc": "Os termos de serviço levantam preocupações muito sérias.",
-    "views.classification.class.none": "Ainda sem nota",
+    "views.classification.class.none": "Ainda não avaliado",
     "views.classification.class_none_desc": "Nós ainda não revisamos os termos.",
     "views.classification.container.text.1": "Os termos de serviço são revisados por contribuidores e divididos em pequenos pontos que podemos discutir, comparar e, enfim, atribuir uma nota com uma insígnia:",
-    "views.classification.container.text.2": "Uma vez que o serviço tem insígnias suficientes para avaliar justeza de seus termos de serviço para os usuários, uma classe é atribuída automaticamente, ponderando as pontuações médias.",
-    "views.classification.container.text.3": "Neste momento, você irá perceber que não há muitos serviços com uma classe atribuída. Isso é porque nós precisamos de mais dados e mais revisões antes que possamos começar a atribui-las. Além disso, ainda estamos experimentado como aplicar as classes.",
+    "views.classification.container.text.2": "Uma vez que o serviço tem insígnias suficientes para avaliar a justeza de seus termos de serviço para os usuários, uma classe é atribuída automaticamente, ponderando as pontuações médias.",
+    "views.classification.container.text.3": "Neste momento, você irá perceber que não existem muitos serviços com uma classe atribuída. Isso é porque nós precisamos de mais dados e mais revisões antes que possamos começar a atribui-las. Além disso, ainda estamos experimentado como aplicar as classes.",
     "views.classification.jumbotron.header": "Classificação",
-    "views.classification.jumbotron.text": "O ToS;DR visa criar um processo transparente e revisado por pares e analisar Termos de Serviço e Políticas de Privacidade para criar uma avaliação da Nota A à Nota E. ",
+    "views.classification.jumbotron.text": "O ToS;DR visa criar um processo transparente e fortemente revisado para avaliar e analisar Termos de Serviço e Políticas de Privacidade a fim de criar uma avaliação que parte da Nota A até a Nota E. ",
     "views.classification.point.bad": "Ruim",
-    "views.classification.point.blocker": "Bloqueante",
+    "views.classification.point.blocker": "Bloqueador",
```

Comment:
Segundo o dicionário, [bloqueador](https://dicionario.priberam.org/bloqueador) e [bloqueante](https://dicionario.priberam.org/bloqueante) são sinônimos. Eu havia escolhido bloqueante por ser um adjetivo de dois gêneros, o que é mais seguro para uma palavra fora de contexto (por exemplo, em algum lugar poderia, em tese, aparecer "Avaliação: bloqueante", mas "Avaliação: bloqueador" seria incorreto já que "bloqueador" só serviria para qualificar substantivos masculinos).

Se for usado somente no contexto do *ponto*, então pode sim ficar *ponto bloqueador*.

Reply:
Nesse ponto não cheguei a comparar as palavras, eu comparei com a tradução de outras línguas mesmo, mas seu ponto é ótimo, talvez seja melhor manter bloqueante mesmo

Reply:
O próprio id da string, `views.classification.point.blocker`, já indica que é para ser usado no contexto de "ponto". Então, para mim, tanto faz aqui.